src/day8/re_frame/trace.cljs

```diff
@@ -234,8 +234,11 @@
             [:thead>tr
              [:th [:button.text-button
                    {:style {:cursor "pointer"}
-                    :on-click #(reset! trace-details {:showing (not (:showing @trace-details))})}
-                   (if (:showing @trace-details) "-" "+")]]
+                    :on-click (fn [e]
+                                (when (:showing @trace-detail-expansions)
+                                  (swap! trace-detail-expansions assoc :overrides {}))
```

Comment:
I will change this part as well. The conditional here is not correct, when expanding or collapsing everything, `:overrides` should always be set to an empty set